README.md

```diff
@@ -20,4 +20,302 @@ composer require bedita/php-sdk
+    $client = new BEditaClient($apiUrl, $apiKey);
+```
+
+### Init Logger for custom log
+
+You can use a custom log, calling `initLogger`:
+
+```php
+   $client->initLogger(['log_file' => '/path/to/file/name.log']);
+```
+
+### Retrieve data
+
+You can use `getObjects(string $type = 'objects', ?array $query = null, ?array $headers = null)` to retrieve a list of objects by type, `getObject(int|string $id, string $type = 'objects', ?array $query = null, ?array $headers = null)` to get a single object by unique identifier and type.
+
+Examples:
+
+```php
+    // get documents
+    $response = (array)$client->getObjects('documents'); // argument passed is a string $type
+
+    // get documents using a filter
+    $query = [
+        'filter' => ['parent' => 'some-parent-id'],
+    ];
+    $response = (array)$client->getObjects('documents', $query); // arguments passed are string $type and array $query
+
+    // get documents using a filter and more query options
+    // include: including related data by relations "has_media" and "part_of"
+    // lang: the desired lang (if translation is available)
+    $query = [
+        'filter' => ['parent' => 'some-parent-id'],
+        'include' => 'has_media,part_of',
+        'lang' => 'en_EN',
+    ];
+    $response = (array)$client->getObjects('documents', $query); // arguments passed are string $type and array $query
+```
+
+```php
+    // get document 17823
+    $response = (array)$client->getObject(17823, 'documents'); // arguments passed are int|string $id, string $type
+
+    // get document 17823, english version, including related data by relations "has_media" and "part_of"
+    $query = [
+        'lang' => 'en_EN',
+        'include' => 'has_media,part_of',
+    ];
+    $response = (array)$client->getObject(17823, 'documents', ['lang' => 'en_EN']); // arguments passed are int|string $id, string $type and array $query
+
+    // get folder website-footer
+    $response = (array)$client->getObject('website-footer', 'folders'); // arguments passed are int|string $id, string $type
+```
+
+You can also use `get(string $path, ?array $query = null, ?array $headers = null)` to retrieve a list of resources or objects or a single resource or object
+
+```php
+    // get api status
+    $response = (array)$client->get('/status'); // argument passed is string $path
+
+    // get api home info
+    $response = (array)$client->get('/home'); // argument passed is string $path
+
+    // get a list of documents
+    $query = [
+        'filter' => ['parent' => 'some-parent-id'],
+    ];
+    $response = (array)$client->get('/documents', $query); // arguments passed are string $path and array $query
+
+    // get a stream by id
+    $response = (array)$client->get('/streams/the-stream-id'); // argument passed is string $path
+```
+
+When you need to get relationships related data, you can you `getRelated(int|string $id, string $type, string $relation, ?array $query = null, ?array $headers = null)`.
```

Comment:
we could probably group methods on relations together in a section like **Working with relations**